coral/src/app/layout/main-navigation/MainNavigation.tsx

```diff
@@ -92,16 +92,17 @@ function MainNavigation() {
           <Divider direction="horizontal" size={2} />
         </Box>
         <li>
-          <MainNavigationSubmenuList icon={settings} text={"Configuration"}>
+          <MainNavigationSubmenuList
+            icon={settings}
+            text={"Configuration"}
+            expanded={pathname.startsWith(Routes.CONFIGURATION)}
```

Comment:
Since this is now added we can make the `expanded` property on `MainNavigationSubmenuList` required and also remove the comment. I thought back then there may be more work to make it work correctly, but passing the prop with`{pathname.startsWith(Routes.CONFIGURATION)}` does it already!

Reply:
Ah yes! I think it might be also good to rename it as `defaultExpanded`, as the initial value we pass by props is overridden by the internal state when clicking the UI.

Reply:
Actually not sure why it should be made mandatory, though!

Reply:
[7a5fa47](https://github.com/Aiven-Open/klaw/pull/1757/commits/7a5fa47e527ca8e555ddc718af10286410ff636d)

Reply:
>To ensure people don't forget to link it to router state?

Yep! If the expanded state is not set by router state, the submenu won't be open when people go directly to this route. And since opening / closing the menu works, it's an easy one to overlook when adding a new submenu. Just one small detail to prevent future devs (and I include myself here 😅 ) from making an easy to avoid mistake.

Reply:
Ok, made it so here ^^ [f6cde9b](https://github.com/Aiven-Open/klaw/pull/1757/commits/f6cde9b037760e7edef25c522e89774a67694763)